src/reducers/toyReducer.js

```diff
@@ -1,35 +1,25 @@
 export const initialState = {
-  toys: []
+  toys: [],
 };
 
 export default function toyReducer(state = initialState, action) {
+  let toys;
```

Comment:
```suggestion
  let toys = [];
```

Ah yeah, makes sense (about the const).  I think what you have is good, only suggestion is you could possible set the default value to be an empty array.  While this doesn't help anything code-wise (since it'll be overridden), it does at least let you know that the intent is that toys will be an array.